manager/manager.go

```diff
@@ -782,6 +782,10 @@ func (m *manager) createContainer(containerName string) error {
 		return err
 	}
 
+	if collectorConfigs != nil {
+		contSpec.HasCustomMetrics = true
+	}
```

Comment:
This doesn't seem to work for the api as we end up calling UpdateSpec() and overwriting this.